src/help/help_impl.cpp

```diff
@@ -918,11 +918,12 @@ void generate_races_sections(const config* help_cfg, section& sec, int level)
 	while(process_queue_again && !taxonomy_queue.empty()) {
 		process_queue_again = false;
 		std::vector<taxonomy_queue_type> to_process = std::move(taxonomy_queue);
+		taxonomy_queue.clear();
```

Comment:
Suggest instead:
```cpp
auto to_process  = std::exchange(taxonomy_queue, {});
```

Reply:
You can use the "suggestion" value for the kind of syntax highlighting to have it be auto-applicable:
```suggestion
		auto to_process = std::exchange(taxonomy_queue, {});
```